stdlib/public/core/StringLegacy.swift

```diff
@@ -34,13 +34,19 @@ extension String {
       return
     }
 
-    // TODO(String performance): We can directly call appendInPlace
-    var result = String()
-    result.reserveCapacity(repeatedValue._guts.count &* count)
-    for _ in 0..<count {
-      result += repeatedValue
+	var repeatedValue = repeatedValue
```

Comment:
Nit: indenting is off; it's too many spaces here and every line until the line after `for i in 0..<count {`.

Reply:
fixed